target_chains/ethereum/contracts/contracts/pyth/Pyth.sol

```diff
@@ -521,6 +581,38 @@ abstract contract Pyth is
         }
     }
 
+    function getPriceFeedsIndex(
```

Comment:
how about `findIndexOfPriceId`?

also this may have been inlined in the other implementation for gas efficiency reasons... @ali-bahjati will know more